README.md

```diff
@@ -1,6 +1,6 @@
 <h1 align="center">
     <a href="https://sylius.com/github-readme/link/" target="_blank">
-        <img src="https://sylius.com/assets/github-readme.png?v110" />
+        <img src="https://sylius.com/assets/github-readme.png?v111" />
```

Comment:
What do you think about having a `github-readme.png?latest` alias on the server? 🤔

Reply:
It's just about refreshing the image quickly. Without a new variable, the image would show delayed

Reply:
@Zales0123 It won't invalidate browser cache.